rcl/test/CMakeLists.txt

```diff
@@ -223,7 +223,7 @@ function(test_target_function)
     APPEND_LIBRARY_DIRS ${extra_lib_dirs}
     INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/../src/rcl/
     LIBRARIES ${PROJECT_NAME} mimick wait_for_entity_helpers
-    AMENT_DEPENDENCIES ${rmw_implementation} "osrf_testing_tools_cpp" "test_msgs"
+    AMENT_DEPENDENCIES ${rmw_implementation} "osrf_testing_tools_cpp" "test_msgs" "rcpputils"
```

Comment:
Technically, it also depends on `rosidl_runtime_c` and `rcutils` as well.

It would also be nice to keep this list in alphabetical order.  But neither of these are blocking, so I'll approve anyway and let you decide whether to fix them.

Reply:
alphabetized and added those two deps as well in e4d8978